src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -97,6 +98,24 @@ public async Task<IEnumerable<CompletionItem>> GetFilteredCompletions(Compilatio
                 .Concat(await moduleReferenceCompletionProvider.GetFilteredCompletions(model.SourceFile.FileUri, context, cancellationToken));
         }
 
+        public static async Task<string> GetResourceDefinitionsMarkdown(string resourceProviderFolder, string apiVersion)
+        {
+            using var client = new HttpClient();
+            client.DefaultRequestHeaders.Add("User-Agent", "Bicep LanguageServer C# App"); // GitHub requires a user agent header
+            var owner = "Azure";
+            var repo = "bicep-types-az";
+            var url = $"https://raw.githubusercontent.com/{owner}/{repo}/main/generated/{resourceProviderFolder}/{apiVersion}/types.md";
```

Comment:
It seems the `resourceProviderFolder` is something like: `keyvault/microsoft.keyvault` or `iothub/microsoft.devices`. Do you know where to get the first part from the current Bicep file?